cpacker/MemGPT
# fix: bug fixing for #1455 - not able to serialize json for Azure

bug fix for this error:

TypeError: Object of type Message is not JSON serializable @ issue #1455

**Please describe the purpose of this pull request.**
Is it to add a new feature? Is it to fix a bug?
- bug fix

**How to test**
How can we test your PR during review? What commands should we run? What outcomes should we expect?
- run with Azure OAI

**Have you tested this PR?**
Have you tested the latest commit on the PR? If so please provide outputs from your tests.
- monkey patched on pymemgpt and it works

**Related issues or PRs**
Please link any related GitHub [issues](https://github.com/cpacker/MemGPT/issues) or [PRs](https://github.com/cpacker/MemGPT/pulls).
https://github.com/cpacker/MemGPT/issues/1455

**Is your PR over 500 lines of code?**
If so, please break up your PR into multiple smaller PRs so that we can review them quickly, or provide justification for its length.
n/a

**Additional context**
Add any other context or screenshots about the PR here.
